server/index.ts

```diff
@@ -26,22 +26,29 @@ app.use(express.static("dist"));
 // Middleware for parsing application/json (strings of the json object is converted into JS)
 app.use(express.json());
 
-// Display all users in MongoDB
+// Display all users from MongoDB
 app.get("/api/users", async (_request, response) => {
   const userCollection = getUserCollection();
   const users = userCollection.find();
   const allUsers = await users.toArray();
   response.send(allUsers);
 });
 
+// Display one specific user from MongoDB
+app.get("/api/users/:name", async (request, response) => {
+  const userCollection = getUserCollection();
+  const user = request.params.name;
+  const isUserKnown = await userCollection.findOne({
```

Comment:
I'd expect `isUserKnown` to be a boolean but it is actually a `user`.
`user` is not really a user but it's the user's name.